apollo/command_to_gcode.py

```diff
@@ -0,0 +1,145 @@
+import asyncio
+
+from apollo.smoothie import SmoothieCom
+from config.settings import Config
+
+
+class CommandToGCode(object): 
+    """
+    this module converts front-end commands to GCode
+    then sends GCode strings to Smoothie-Com module
+    """
+
+    def __init__(self):
+        self.smoothie_com = SmoothieCom(Config.SMOOTHIE_URL, Config.SMOOTHIE_PORT)
+
+    @asyncio.coroutine
+    def connect(self):
+        """
+        tell the smoothie module to connect to the motor controller
+        return True if successful
+        """
+        return (yield from self.smoothie_com.connect())
+
+    @asyncio.coroutine
+    def send_gcode(self,string):
+        """
+        send a GCode string to the smoothie module
+        """
+        if len(string):
```

Comment:
I believe `if string` is equivalent to if `if len(string)` since python non-empty strings evaluate to truthy..

Also, `string` isn't too meaningful by itself, maybe `gcode_str` .. or even just `gcode` and  you could indicate the type in the doc string... If you really wanted to you can use python 3.4 type hinting,

```python
def send_gcode(self, gcode : str):
....
```